app/src/App.css

```diff
@@ -0,0 +1,82 @@
+.App {
```

Comment:
Feel free to delete the boilerplate CSS here so it doesn't get in the way during reviews, etc.